pkg/controllers/managementuser/rbac/namespace_handler.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/rancher/norman/types/convert"
 	"github.com/rancher/norman/types/slice"
 	"github.com/rancher/rancher/pkg/apis/management.cattle.io"
-	apisV3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	v32 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
```

Comment:
why did you change it to `v32`? I think `apisv3` looks better, and it is consistent with other parts of the code

Reply:
I'll fix it asap